src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -33,3 +33,4 @@ CommentStep.Descriptor.DisplayName=JIRA: Add a comment to issue(s)
 SearchIssuesStep.Descriptor.DisplayName=JIRA: Search issues
 IssueSelectorStep.Descriptor.DisplayName=JIRA: Issue selector
 JiraCreateIssueNotifier.DisplayName=JIRA: Create issue
+FindIssuesAtCommitLogsStep.Descriptor.DisplayName=JIRA: find all issues referenced by the commit logs of this build
```

Comment:
Ah, that's interesting, but I'd like to group all dropdown options together (i.e. not only workflow ones). Can this be done?